client/src/components/ui/button.tsx

```diff
@@ -18,6 +18,7 @@ const buttonVariants = cva(
           "bg-secondary text-secondary-foreground hover:bg-secondary/80",
         ghost: "hover:bg-accent hover:text-accent-foreground",
         link: "text-primary underline-offset-4 hover:underline",
+        signup: "bg-[#768970] text-l text-white hover:bg-[#6A7C5E]",
```

Comment:
Might be a little excessive to have a 'signup' variant in the button. Looks like it is only used in two places, think it's fine to just duplicate.

Additionally could you use one of the tailwind theme color for the background?